### PR TITLE
debugging jest using vscode doc

### PR DESCRIPTION
**Pull request by cwatsonc**
#### Checklist

adding a WIP for debugging documents -- will adjust based on team feedback

### CODE REVIEW

File: web/docs/DebuggingJest.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+I found the following techniques helpful in understanding vscode debugging integration with jest test modules.  There are fancier vscode extensions (e.g. vscode-jest) that worked slick, but were a lot harder to understand in configuration -- this method uses a seperate launch configuration that you can execute from the debug console on the open test. Add this fragment to the launch.json inside the configurations property.
+
+It's worth noting that I struggled with the workspace configurations until I mounted the workspace at sub-project folder `FreeBoardgames.org/web` making web the root of the workspace. 
+
+[Article Describing Jest Debug using VSCode](https://www.basefactor.com/using-visual-studio-code-to-debug-jest-based-unit-tests)
+
+
+
+```JSON
+/** launch.json **/
```
Comment (flamecoals) [inline web/docs/DebuggingJest.stories.mdx:18]: wouldn't it be easier to commit this to  the repo (without restricting it to a single game, or commiting one for  each game...) and just point out here what menus the developers need to go on VSCode to point  out?

Btw,  thanks for this, I use VSCode too :)

Comment (cwatsonc) [reply web/docs/DebuggingJest.stories.mdx:18]: @flamecoals not sure if you're awaiting some input?  My feeling is this type of config shouldn't be checked into code lines as it pollutes unnecessarily -- ever wrestled with a team wanting specific linters, code checks?  But this stuff should be close to project docs if project is to have a healthy community commitment.  I always appreciate devs insulating the team from their specific choices, but sharing them so we can develop better skills and/or tools.

Comment (flamecoals) [reply web/docs/DebuggingJest.stories.mdx:18]: Sure, sounds good then. You can merge the docs :)

**Review by flamecoals — APPROVED** [review]

**Comment by flamecoals** [comment]
It was, we need to push to prod so it can be seen in the docs page. I am
traveling so I don't have access to my normal computer with the keys to do
so, but Jason does. I am asking him to push to prod today.

**Pull request merged**